tests/fixtures/current_user_capitalized_context_types.json

```diff
@@ -0,0 +1,152 @@
+{
+	"add_favorite_course": {
+		"method": "POST",
+		"endpoint": "users/self/favorites/courses/1",
+		"data":
+			{
+				"context_id": 1,
+				"context_type": "Course"
+			},
+		"status_code": 200
+	},
+	"add_favorite_course_by_sis_id": {
+		"method": "POST",
+		"endpoint": "users/self/favorites/courses/sis_course_id:test-sis-id",
+		"data":
+			{
+				"context_id": 1,
+				"context_type": "Course"
+			},
+		"status_code": 200
+	},
+	"add_favorite_group": {
+		"method": "POST",
+		"endpoint": "users/self/favorites/groups/1",
+		"data":
+			{
+				"context_id": 1,
+				"context_type": "Group"
+			},
+		"status_code": 200
+	},
+	"add_favorite_group_by_sis_id": {
+		"method": "POST",
+		"endpoint": "users/self/favorites/groups/sis_group_id:test-sis-id",
+		"data":
+			{
+				"context_id": 1,
+				"context_type": "Group"
+			},
+		"status_code": 200
+	},
+	"get_by_id": {
+		"method": "GET",
+		"endpoint": "users/self",
+		"data": {
+			"id": 1,
+			"name": "John Doe"
+		},
+		"status_code": 200
+	},
```

Comment:
I believe this endpoint is pretty much unchanged from the one that's already in `current_user.json`, so we can just reuse that one.